dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/MessageConversationController.java

```diff
@@ -379,6 +380,10 @@ public WebMessage postMessageConversationReply(
       throw new AccessDeniedException("Not authorized to send internal messages");
     }
 
+    if (!hasAccessToMessageConversation(currentUser, conversation)) {
+      throw new AccessDeniedException("Not authorized to send messages to this conversation.");
```

Comment:
Could we use the https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ForbiddenException.java instead?